src/iop/levels.c

```diff
@@ -857,10 +857,11 @@ static gboolean dt_iop_levels_area_draw(GtkWidget *widget, cairo_t *crf, gpointe
   // draw grid
   cairo_set_line_width(cr, DT_PIXEL_APPLY_DPI(.4));
   cairo_set_source_rgb(cr, .1, .1, .1);
-  if(dev->histogram_type == DT_DEV_HISTOGRAM_WAVEFORM)
-    dt_draw_waveform_lines(cr, 0, 0, width, height);
-  else
-    dt_draw_vertical_lines(cr, 4, 0, 0, width, height);
+  // NOTE: if we actually draw a waveform in the background, we might want this:
```

Comment:
I would just remove this commented out code:
- level is deprecated and one should prefer the rgblevel iop
- I don't think we will ever draw a waveform in any iop backgroud

Reply:
Sounds fair. Fixed in #4140, where this part of the code has migrated.